cluster/engine_test.go

```diff
@@ -156,10 +158,12 @@ func TestEngineCpusMemory(t *testing.T) {
 	apiClient := engineapimock.NewMockClient()
 	apiClient.On("Info", mock.Anything).Return(mockInfo, nil)
 	apiClient.On("ServerVersion", mock.Anything).Return(mockVersion, nil)
+	apiClient.On("NetworkList", mock.Anything,
+		mock.AnythingOfType("NetworkListOptions"),
+	).Return([]types.NetworkResource{}, nil)
+	apiClient.On("VolumeList", mock.Anything, mock.Anything).Return(types.VolumesListResponse{}, nil)
```

Comment:
No. VolumeList takes a generic filters.Args. NetworksList has a specific struct that it takes. 

I can try ` mock.AnythingOfType("Args")` but that might be less helpful than just being `Anything`. What do you think?